_spinnaker/pacrd.md

```diff
@@ -53,39 +53,56 @@ accounts
 
 ## Quick Start
 
-This quick start assumes the following about your Kubernetes cluster:
-
-- Your Spinnaker cluster is installed into a namespace called `spinnaker`
-- Your Spinnaker services have `service` objects of the form `spin-{service_name}`; for example, `spin-front50`
-
 Download the current `pacrd` manifest to your local machine:
 
 ```
-curl -fsSL https://engineering.armory.io/manifests/pacrd-0.1.0.yaml > pacrd-0.1.0.yaml
+curl -fsSL https://engineering.armory.io/manifests/pacrd-0.1.1.yaml > pacrd-0.1.1.yaml
 ```
 
 Then, inspect the manifest to make sure it is compatible with your cluster.
 
-When you're ready, apply the `pacrd` manifest to your cluster:
+Create the following files in the directory where you downloaded the `pacrd` manifest to customize the
```

Comment:
This said `current directory`. I changed it to the manifest directory cause I think that's what you meant. Feel free to update it to something more specific.